packages/flutter/lib/src/gestures/details_with_positions.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show Offset;
+
+import 'package:flutter/foundation.dart'
+    show DiagnosticPropertiesBuilder, Diagnosticable, DiagnosticsProperty;
+
+/// Details that contain positions at which the single pointer interacts
+/// with the screen.
```

Comment:
This doc isn't very informative as it basically just blows the name of the class up into a sentence, see also https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-useless-documentation.

See also https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#writing-prompts-for-good-documentation for some prompts on writing better documentation.

Reply:
Updated.